apps/test/unit/templates/teacherDashboard/SectionAssignerTest.js

```diff
@@ -23,19 +24,34 @@ describe('SectionAssigner', () => {
         initialSelectedSectionId={fakeTeacherSectionsForDropdown[0].id}
         sections={fakeTeacherSectionsForDropdown}
         selectSection={() => {}}
+        showAssignButton={true}
       />
     );
 
     expect(wrapper.find('AssignButton').exists()).to.be.true;
   });
 
+  it('does not render an AssignButton if showAssignButton is false', () => {
+    const wrapper = mount(
+      <SectionAssigner
+        initialSelectedSectionId={fakeTeacherSectionsForDropdown[0].id}
+        sections={fakeTeacherSectionsForDropdown}
+        selectSection={() => {}}
+        showAssignButton={false}
```

Comment:
at some point you may want to extract "default" props, to make it easier to see what's changing between test cases: 

https://github.com/code-dot-org/code-dot-org/blob/54530b4070901065a13c3354ca2bd019de0e5cdc/apps/test/unit/lib/ui/accounts/DeleteAccountTest.jsx#L11-L16

https://github.com/code-dot-org/code-dot-org/blob/54530b4070901065a13c3354ca2bd019de0e5cdc/apps/test/unit/lib/ui/accounts/DeleteAccountTest.jsx#L53-L55